KuthumiPepple/Speedy-Loans
# #166102174 Create logo and add license badge to README.md

#### What does this PR do?
Have Speedy Loan's logo and MIT license badge on README.md
#### Description of Task to be completed?
Add image title and image link to README.md
Add badge URL; add URL of LICENSE file to badge
#### How should this be manually tested?
- fetch PR
- checkout branch
- open README in browser with Markdown viewer extension
- You should see both logo and license badge. Click on badge to view LICENSE
#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
#166102174
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30692765/57982715-1aa51480-7a41-11e9-9ccd-fc458356b86d.png)
#### Questions:
Nil
